sidechain/src/tests.rs

```diff
@@ -39,6 +39,7 @@ fn confirm_imported_sidechain_block_works_for_correct_shard() {
 			Origin::signed(signer7.clone()),
```

Comment:
I would like to see some asserts about the `SidechainBlockFinalizationCandidate` here.